marine.h

```diff
@@ -21,6 +21,50 @@ void marine_free(marine_result *ptr);
 void marine_report_fields(void);
 void destroy_marine(void);
 
+
+enum marine_packet_field_value_type {
+    MARINE_VT_NONE,
+    MARINE_VT_INT,
+    MARINE_VT_UINT,
+    MARINE_VT_FLOAT,
+    MARINE_VT_BOOL,
+    MARINE_VT_STR,
+    MARINE_VT_BYTES,
+    MARINE_VT_LIST
+};
+
+typedef struct _marine_packet_field_value {
+    union {
+        double float_value;
+        int int_value;
+        unsigned int uint_value;
+        unsigned char bool_value;
+        char *str_value;
+        struct _marine_packet_field_value *list_value;
+    };
+    unsigned int len;
+    enum marine_packet_field_value_type type;
+} marine_packet_field_value;
+
+typedef struct {
+    char *name;
+
```

Comment:
Why the empty line? Also in `marine_packet`

Reply:
Removed.